app/templates/components/events/view/export/download-common.hbs

```diff
@@ -0,0 +1,21 @@
+<h2 class="ui header">
+  <div class="content">
+    {{t 'Download Event as'}} {{downloadType}}
+  </div>
+  <div class="sub header">
```

Comment:
Why is everything inside the header ? This doesn't make sense. Move it outside the header.